### PR TITLE
Created a new feature FlushConnection for Network Security Groups.

### PR DESCRIPTION
**Pull request by Satya-anshu**
## Description

Added FlushConnection feature to NSG.
Cmdlet Design link: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1167

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Network/Network/NetworkSecurityGroup/NewAzureNetworkSecurityGroupCommand.cs [hunk 1]
```diff
@@ -52,6 +52,12 @@ public class NewAzureNetworkSecurityGroupCommand : NetworkSecurityGroupBaseCmdle
         [ValidateNotNullOrEmpty]
         public virtual string Location { get; set; }
 
+        [Parameter(
+             Mandatory = false,
+             ValueFromPipelineByPropertyName = true,
+             HelpMessage = "When enabled, flows created from Network Security Group connections will be re-evaluated when rules are updates. Initial enablement will trigger re-evaluation")]
```
Comment (isra-fel) [inline src/Network/Network/NetworkSecurityGroup/NewAzureNetworkSecurityGroupCommand.cs:58]:
```suggestion
             HelpMessage = "When enabled, flows created from Network Security Group connections will be re-evaluated when rules are updated. Initial enablement will trigger re-evaluation")]
```

File: src/Network/Network/help/New-AzNetworkSecurityGroup.md [hunk 2]
```diff
@@ -142,6 +149,22 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -FlushConnection
+When enabled, flows created from Network Security Group connections will be re-evaluated when rules are updates.
```
Comment (isra-fel) [inline src/Network/Network/help/New-AzNetworkSecurityGroup.md:153]:
```suggestion
When enabled, flows created from Network Security Group connections will be re-evaluated when rules are updated.
```

File: src/Network/Network/Network.format.ps1xml [hunk 3]
```diff
@@ -2104,10 +2104,14 @@
                 <Label>ProvisioningState</Label>
                 <PropertyName>ProvisioningState</PropertyName>
               </ListItem>
-              <ListItem>
-                <Label>Tags</Label>
-                <PropertyName>TagsTable</PropertyName>
-              </ListItem>
+			  <ListItem>
```
Comment (isra-fel) [inline src/Network/Network/Network.format.ps1xml:2107]: Please use spaces instead of tabs to keep the consistency of indentation this file

**Review by isra-fel — COMMENTED** [review]
Generally LGTM

**Review by isra-fel — APPROVED** [review]

**Comment by MikhailTryakhov** [comment]
@isra-fel  yes we need this pattern due to similar naming style with swagger

**Review by isra-fel — APPROVED** [review]

**Pull request merged**